### ISSUE TITLE
Paywall: give the list of available domains

### ISSUE THREAD

**Issue by gnppn** [wallabag/wallabag#3434]
### Issue details

When adding new site credentials, you have to manually enter the domain in the ~/site-credentials/new page. Wallabag doesn't give any clue on the supported ones.

A solution could be to have a scrollable list of supported domains or a documentation page referencing them for the user, with a way to request the addition of another.

### Environment

Wallabag.it version 2.3

### Steps to reproduce/test case

Add new site credentials for a paywall.

**Comment by bdunogier** [comment]
How much can be customize the editing pages for settings ? We could check the submitted domain against graby, and see if `requires_authentication` is enabled for the site.

Having a list of paywall domains would probably require that we scan & load, on container compilation, the domains that have this flag.

**Comment by j0k3r** [comment]
Yeah or maybe we can just write them down in a configuration file in wallabag. It'll save a lot of CPU time :slightly_smiling_face:
